packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -211,6 +217,8 @@ class DropdownButton extends React.Component {
           ref={this.setButtonNode}
           aria-expanded={isOpen}
           aria-label={buttonAriaLabel}
+          aria-haspopup="true"
+          aria-controls={dropDownMenuListId}
```

Comment:
we need to add aria-controls for split button as well : https://github.com/cerner/terra-core/blob/7cb2fd38b0daa2deb8fb1a72d49d9502c0fdbebe/packages/terra-dropdown-button/src/SplitButton.jsx#L281

Reply:
Implemented.